code/switch.js

```diff
@@ -0,0 +1,295 @@
+          alert (`You chose Bolognese.`)
+        break
+        case 3:
+          alert (`You chose Alfredo.`)
+        break
+        default:
+          alert (`You chose nothing.`)
+          exit (1)
+      }
+    break
+    case 3:
+      switch (choice) {
+        case 1:
+          alert (`You chose with meet.`)
+        break
+        case 2:
+          alert (`You chose vegetarian.`)
+        break
+        case 3:
+          alert (`You chose vegan.`)
+        break
+        default:
+          alert (`You chose nothing.`)
+          exit(1)
+      }
+    break
+    default:
+      alert (`You chose nothing.`)
+      exit(1)
+  } 
+
+  console.log (choice)
+  
+  
+  // Step 4 - Age
+  // Your code goes here
+  
+  
+  const age = parseInt 
+  (prompt (`Do you want to order the child menu or the adult menu. Please type your age:`))
+
+  console.log(typeof age)
+  
+  
+  const foodName = ["Pizza", "Pasta", "Lasagne"]
+  const subtypeName = [
+    ["Margherita", "Pepperoni", "Hawaiian"],
+    ["Carbonara", "Bolognese", "Alfredo"],
+    ["with meet", "vegetarian", "vegan"]
+  ]
+  
+  const foodResult = foodName [food - 1]
+  
+  const subtypeResult = subtypeName [food - 1] [choice - 1]
+  
+  switch (age) {
+    case 100:
+    case 99:
+    case 98:
+    case 97:
+    case 96:
+    case 95:
+    case 94:
+    case 93:
+    case 92:
+    case 91:
+    case 90:
+    case 89:
+    case 88:
+    case 87:
+    case 86:
+    case 85:
+    case 84:
+    case 83:
+    case 82:
+    case 81:
+    case 80:
+    case 79:
+    case 78:
+    case 77:
+    case 76:
+    case 75:
+    case 74:
+    case 73:
+    case 72:
+    case 71:
+    case 70:
+    case 69:
+    case 68:
+    case 67:
+    case 66:
+    case 65:
+    case 64:
+    case 63:
+    case 62:
+    case 61:
+    case 60:
+    case 59:
+    case 58:
+    case 57:
+    case 56:
+    case 55:
+    case 54:
+    case 53:
+    case 52:
+    case 51:
+    case 50:
+    case 49:
+    case 48:
+    case 47:
+    case 46:
+    case 45:
+    case 44:
+    case 43:
+    case 42:
+    case 41:
+    case 40:
+    case 39:
+    case 38:
+    case 37:
+    case 36:
+    case 35:
+    case 34:
+    case 33:
+    case 32:
+    case 31:
+    case 30:
+    case 29:
+    case 28:
+    case 27:
+    case 26:
+    case 25:
+    case 24:
+    case 23:
+    case 22:
+    case 21:
+    case 20:
+    case 19:
+    case 18:
+      alert (`We will prepare one adult size ${foodResult} ${subtypeResult} for you. That'll cost 20 CHF in total. `)
+    break
+    case 17:
+    case 16:
+    case 15:
+    case 14:
+    case 13:
+    case 12:
+    case 11:
+    case 10:
+    case 9:
+    case 8:
+    case 7:
+    case 6:
+    case 5:
+    case 4:
+    case 3:
+    case 2:
+    case 1:
+      alert (`We will prepare one child size ${foodResult} ${subtypeResult} for you. That'll cost 15 CHF in total.`)
+    break
+    default:
+      alert (`You didn't enter a real number. Please try again.`)
+      exit (1)
+  }
```

Comment:
this statement works, but it is not the correct approach to check this case. Consider this:

```
switch (true) {
  case age >= 1 && age <= 18:
    alert(`We will prepare one child size ${foodResult} ${subtypeResult} for you. That'll cost 15 CHF in total.`);
    break;
  case age >= 19 && age <= 100:
    alert(`We will prepare one adult size ${foodResult} ${subtypeResult} for you. That'll cost 20 CHF in total.`);
    break;
  default:
    alert(`You didn't enter a real number. Please try again.`);
}
```
👍